db/schema.rb

```diff
@@ -2,8 +2,8 @@
 # of editing this file, please use the migrations feature of Active Record to
 # incrementally modify your database, and then regenerate this schema definition.
 #
-# This file is the source Rails uses to define your schema when running `rails
-# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# This file is the source Rails uses to define your schema when running `bin/rails
```

Comment:
Please exclude the changes to schema.rb from this PR.

Reply:
Hello, I have excluded the changes in **db/schema.rb** and also changed the **qqq.yml** file.